src/components/button/Button.test.ts

```diff
@@ -0,0 +1,50 @@
+import { render } from '../../utils/domHandlers';
+import { Button } from './Button';
+
+describe('Test Button Component', () => {
+  beforeEach(() => {
+    document.body.innerHTML = '<div id="geo-app"></div>';
+  });
+
+  it('Should render text correctly', () => {
+    const appComponent = document.getElementById('geo-app');
+    appComponent &&
+      render(
+        Button('text', () => {}),
+        appComponent
+      );
+    expect(document.getElementById('button')?.innerText).toBe('text');
+  });
+
+  it('Should contain className', () => {
+    const appComponent = document.getElementById('geo-app');
+
+    appComponent &&
+      render(
+        Button('text', () => {}, 'false-btn'),
+        appComponent
```

Comment:
Tutaj byśmy musieli refaktor zrobić. Trochę brzydko to wygląda

Reply:
Chodzi o "&&" ? Lubię ten zapis bo wykonuje mi drugą część kodu, omijając korzystanie z if-ki czy ternary operator. Zapis jest krótszy i moim zdaniem czytelniejszy. :)

Reply:
Miałem na myśli refaktor montowania komponentu do testu. Lepszym pomysłem będzie napisanie jednej funkcji, która będzie tworzyła element i podpinała button z przekazanymi properties, zamiast pisać w każdym miejscu appComponent && ...

Reply:
aaaa, ok :) Poprawione